exec/infrastructure/positionmanager/entrypoint.py

```diff
@@ -5,7 +5,7 @@
 
 import requests
 from loguru import logger
-from pocketsizefund.config import config
+from pocketsizefund import config
```

Comment:
This broke the tests without the `pocketsizefund.config` import.

Reply:
fixed